loop-server/manifest.json

```diff
@@ -2,16 +2,30 @@
   "envs" : {
     "stage": {
       "test_types": [ "stack-check", "loadtest" ],
+      "stack_label": "loopsvrstg",
+      "aws_region": "eu-west-1",
       "urls": {
 	"root" : "loop.stage.mozaws.net",
-	"push_config" : "loop.stage.mozaws.net/push-server-config"
+	"push_config" : "loop.stage.mozaws.net/push-server-config",
+	"heartbeat" : "loop.stage.mozaws.net/__heartbeat__"
+      }
+    },
+    "pre-prod": {
+      "test_types": [ "stack-check" ],
+      "aws_region": "eu-west-1",
+      "urls": {
+        "root": "loop.services.mozilla.com",
+	"push_config" : "loop.services.mozilla.com/push-server-config",
+	"heartbeat" : "loop.services.mozilla.com/__heartbeat__"
       }
     },
     "prod": {
       "test_types": [ "stack-check" ],
+      "aws_region": "eu-west-1",
       "urls": {
         "root": "loop.services.mozilla.com",
-	"push_config" : "loop.services.mozilla.com/push-server-config"
+	"push_config" : "loop.services.mozilla.com/push-server-config",
+	"heartbeat" : "loop.services.mozilla.com/__heartbeat__"
```

Comment:
Re: `push_config` and `heartbeat`,

Since these endpoints use the same `root` urls, not sure if we want/need to specify the entire URI, or just the path.

For example:
```js
"urls": {
  "root": "loop.services.mozilla.com",
  "push_config" : "loop.services.mozilla.com/push-server-config",
  "heartbeat": "loop.services.mozilla.com/__heartbeat__"
}
```

versus:
```
"urls": {
  "root": "loop.services.mozilla.com",
  "push_config" : /push-server-config",
  "heartbeat": "/__heartbeat__"
}
```

... And then in our code, we just concatenate `{{ urls.root + urls.push_config }}`. That way we aren't repeating the `urls.root` 3x and having potential typos or copy-pasta errors.

Conversely, maybe something like:
```
"push_config": "{{ urls.root }}/push-server-config",
``` 

Not sure how feasible any of that is, but something about duplication of things makes me scared.